app/assets/v2/js/pages/bounty_details.js

```diff
@@ -578,6 +578,8 @@ var do_actions = function(result) {
     var should_block_from_starting_work = !is_interested && result['project_type'] == 'traditional' && (result['status'] == 'started' || result['status'] == 'submitted');
     var show_start_stop_work = is_still_on_happy_path && !should_block_from_starting_work;
     var show_github_link = result['github_url'].substring(0, 4) == 'http';
+    var show_job_description = result['attached_job_description'] && result['attached_job_description'].substring(0, 4) == 'http';
```

Comment:
update this to `startsWith('http')` ? makes code more redable

Reply:
updated